### ISSUE TITLE
Updating td-agent-bit erase current configuration file without asking (CentOS 7)

### ISSUE THREAD

**Issue by Theoooooo** [fluent/fluent-bit#1484]
## Bug Report

**Describe the bug**
Each time i upgrade the package present on the repo ``` http://packages.fluentbit.io/centos/7 ```, it erase the files td-agent-bit.conf and parsers.conf with the default file.

**To Reproduce**

1. Install an older version than the last one available on the repo above
2. Modify on or both the configuration files by any way
3. Uptade the version with ``` yum update ``` and ``` y ``` to confirm the upgrade
4. After the update the service does not restart so it keep the last loaded configuration file in memory but it has replaced the 2 files with the default one.
5. Restarting the service reload the default configuration file that have replaced your old configuration file.

**Expected behavior**
If you could update your post-installation script, that would be helpfull because it's not the first time it's happening for me. It's like the third time.

**Your Environment**
CentOS: 7.6
repo : 
```
[td-agent-bit]
name = TD Agent Bit
baseurl = http://packages.fluentbit.io/centos/7
gpgcheck=1
gpgkey=http://packages.fluentbit.io/fluentbit.key
enabled=1
 ```

**Comment by wiza** [comment]
We are seeing this too, really annoying and unexpected.

**Comment by Theoooooo** [comment]
Any news on this ?
It's like something really annoying and it should be fixed quickly, i don't know how many peoples are using it on CentOS but i think it has bothered more peoples than i expected ...

**Comment by edsiper** [comment]
thanks for reporting the problem. This is already fixed in v1.3 release:

```
[root@88159dfee38b /]# rpm -i --force -vv /rpm/td-agent-bit-1.3.1-1.x86_64.rpm 
D: ============== /rpm/td-agent-bit-1.3.1-1.x86_64.rpm
D: loading keyring from pubkeys in /var/lib/rpm/pubkeys/*.key
D: couldn't find any keys in /var/lib/rpm/pubkeys/*.key
D: loading keyring from rpmdb
D: opening  db environment /var/lib/rpm cdb:0x401
D: opening  db index       /var/lib/rpm/Packages 0x400 mode=0x0
D: locked   db index       /var/lib/rpm/Packages
D: opening  db index       /var/lib/rpm/Name 0x400 mode=0x0
D:  read h#     172 Header SHA1 digest: OK (489efff35e604042709daf46fb78611fe90a75aa)
D: added key gpg-pubkey-f4a80eb5-53a7ff4b to keyring
D:  read h#     309 Header SHA1 digest: OK (dd737a402556b7653c2bc971f343532046e26384)
D: added key gpg-pubkey-352c64e5-52ae6884 to keyring
D: Using legacy gpg-pubkey(s) from rpmdb
D: Expected size:      6269788 = lead(96)+sigs(180)+pad(4)+data(6269508)
D:   Actual size:      6269788
D: /rpm/td-agent-bit-1.3.1-1.x86_64.rpm: Header SHA1 digest: OK (8bf370556213cf27a973956ecfa1bdf8f87412ca)
D: 	added binary package [0]
D: found 0 source and 1 binary packages
D: opening  db index       /var/lib/rpm/Conflictname 0x400 mode=0x0
D: ========== +++ td-agent-bit-1.3.1-1 x86_64/linux 0x2
D: opening  db index       /var/lib/rpm/Basenames 0x400 mode=0x0
D:  read h#     176 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: /bin/sh                                       YES (db files)
D:  Requires: /bin/sh                                       YES (cached)
D:  Requires: /bin/sh                                       YES (cached)
D:  Requires: /bin/sh                                       YES (cached)
D:  Requires: config(td-agent-bit) = 1.3.1-1                YES (added provide)
D: opening  db index       /var/lib/rpm/Providename 0x400 mode=0x0
D:  read h#     179 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: ld-linux-x86-64.so.2()(64bit)                 YES (db provides)
D:  Requires: ld-linux-x86-64.so.2(GLIBC_2.3)(64bit)        YES (db provides)
D:  Requires: libc.so.6()(64bit)                            YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.10)(64bit)                  YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.14)(64bit)                  YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.16)(64bit)                  YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.17)(64bit)                  YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.2.5)(64bit)                 YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.3)(64bit)                   YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.3.2)(64bit)                 YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.3.4)(64bit)                 YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.4)(64bit)                   YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.6)(64bit)                   YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.7)(64bit)                   YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.8)(64bit)                   YES (db provides)
D:  Requires: libc.so.6(GLIBC_2.9)(64bit)                   YES (db provides)
D:  read h#     188 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: libcrypto.so.10()(64bit)                      YES (db provides)
D:  Requires: libcrypto.so.10(libcrypto.so.10)(64bit)       YES (db provides)
D:  Requires: libdl.so.2()(64bit)                           YES (db provides)
D:  Requires: libdl.so.2(GLIBC_2.2.5)(64bit)                YES (db provides)
D:  read h#     173 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: libgcc_s.so.1()(64bit)                        YES (db provides)
D:  Requires: libgcc_s.so.1(GCC_3.0)(64bit)                 YES (db provides)
D:  Requires: libgcc_s.so.1(GCC_3.3)(64bit)                 YES (db provides)
D:  Requires: libgcc_s.so.1(GCC_4.2.0)(64bit)               YES (db provides)
D:  Requires: libm.so.6()(64bit)                            YES (db provides)
D:  Requires: libm.so.6(GLIBC_2.2.5)(64bit)                 YES (db provides)
D:  Requires: libpthread.so.0()(64bit)                      YES (db provides)
D:  Requires: libpthread.so.0(GLIBC_2.12)(64bit)            YES (db provides)
D:  Requires: libpthread.so.0(GLIBC_2.2.5)(64bit)           YES (db provides)
D:  Requires: libpthread.so.0(GLIBC_2.3.2)(64bit)           YES (db provides)
D:  Requires: libpthread.so.0(GLIBC_2.3.4)(64bit)           YES (db provides)
D:  Requires: librt.so.1()(64bit)                           YES (db provides)
D:  Requires: librt.so.1(GLIBC_2.2.5)(64bit)                YES (db provides)
D:  Requires: libssl.so.10()(64bit)                         YES (db provides)
D:  Requires: libssl.so.10(libssl.so.10)(64bit)             YES (db provides)
D:  read h#     225 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: libsystemd.so.0()(64bit)                      YES (db provides)
D:  Requires: libsystemd.so.0(LIBSYSTEMD_209)(64bit)        YES (db provides)
D:  read h#      14 Header V3 RSA/SHA256 Signature, key ID f4a80eb5: OK
D:  Requires: libz.so.1()(64bit)                            YES (db provides)
D:  Requires: libz.so.1(ZLIB_1.2.0)(64bit)                  YES (db provides)
D:  Requires: libz.so.1(ZLIB_1.2.2)(64bit)                  YES (db provides)
D:  Requires: rpmlib(CompressedFileNames) <= 3.0.4-1        YES (rpmlib provides)
D:  Requires: rpmlib(FileDigests) <= 4.6.0-1                YES (rpmlib provides)
D:  Requires: rpmlib(PayloadFilesHavePrefix) <= 4.0-1       YES (rpmlib provides)
D:  Requires: rtld(GNU_HASH)                                YES (db provides)
D:  Requires: rpmlib(PayloadIsXz) <= 5.2-1                  YES (rpmlib provides)
D: opening  db index       /var/lib/rpm/Obsoletename 0x400 mode=0x0
D: ========== recording tsort relations
D: ========== tsorting packages (order, #predecessors, #succesors, depth)
D:     0    0    0    1   +td-agent-bit-1.3.1-1.x86_64
D: installing binary packages
D: Selinux disabled.
D: closed   db index       /var/lib/rpm/Obsoletename
D: closed   db index       /var/lib/rpm/Conflictname
D: closed   db index       /var/lib/rpm/Providename
D: closed   db index       /var/lib/rpm/Basenames
D: closed   db index       /var/lib/rpm/Name
D: closed   db index       /var/lib/rpm/Packages
D: closed   db environment /var/lib/rpm
D: opening  db environment /var/lib/rpm cdb:0x401
D: opening  db index       /var/lib/rpm/Packages (none) mode=0x42
D: sanity checking 1 elements
D: running pre-transaction scripts
D: computing 9 file fingerprints
D: opening  db index       /var/lib/rpm/Name (none) mode=0x42
D: opening  db index       /var/lib/rpm/Basenames (none) mode=0x42
D: opening  db index       /var/lib/rpm/Group (none) mode=0x42
D: opening  db index       /var/lib/rpm/Requirename (none) mode=0x42
D: opening  db index       /var/lib/rpm/Providename (none) mode=0x42
D: opening  db index       /var/lib/rpm/Conflictname (none) mode=0x42
D: opening  db index       /var/lib/rpm/Obsoletename (none) mode=0x42
D: opening  db index       /var/lib/rpm/Triggername (none) mode=0x42
D: opening  db index       /var/lib/rpm/Dirnames (none) mode=0x42
D: opening  db index       /var/lib/rpm/Installtid (none) mode=0x42
D: opening  db index       /var/lib/rpm/Sigmd5 (none) mode=0x42
D: opening  db index       /var/lib/rpm/Sha1header (none) mode=0x42
Preparing packages...
D: computing file dispositions
D: 0x00000045     4096    143953577     57560164 /
D: ========== +++ td-agent-bit-1.3.1-1 x86_64-linux 0x2
D: Expected size:      6269788 = lead(96)+sigs(180)+pad(4)+data(6269508)
D:   Actual size:      6269788
D: td-agent-bit-1.3.1-1.x86_64: Header SHA1 digest: OK (8bf370556213cf27a973956ecfa1bdf8f87412ca)
D:   install: td-agent-bit-1.3.1-1 has 9 files
D:  read h#     321 Header SHA1 digest: OK (8bf370556213cf27a973956ecfa1bdf8f87412ca)
D: %pre(td-agent-bit-1.3.1-1.x86_64): scriptlet start
D: %pre(td-agent-bit-1.3.1-1.x86_64): execv(/bin/sh) pid 32
D: %pre(td-agent-bit-1.3.1-1.x86_64): waitpid(32) rc 32 status 0
td-agent-bit-1.3.1-1.x86_64
D: ========== Directories not explicitly included in package:
D:          0 /etc/
D:          2 /lib/systemd/system/
D:          3 /lib64/
D:          4 /opt/
D: ==========
D: create     040755  2 (   0,   0)     0 /etc/td-agent-bit
D: create     100644  1 (   0,   0)  4038 /etc/td-agent-bit/parsers.conf;5d976b9e
D: create     100644  1 (   0,   0)    45 /etc/td-agent-bit/plugins.conf;5d976b9e
D: skip       100644  1 (   0,   0)  1002 /etc/td-agent-bit/td-agent-bit.conf;5d976b9e
D: create     100644  1 (   0,   0)   237 /lib/systemd/system/td-agent-bit.service;5d976b9e
D: create     100755  1 (   0,   0)17811792 /lib64/libfluent-bit.so;5d976b9e
D: create     040755  3 (   0,   0)     0 /opt/td-agent-bit
D: create     040755  2 (   0,   0)     0 /opt/td-agent-bit/bin
D: create     100755  1 (   0,   0)6562448 /opt/td-agent-bit/bin/td-agent-bit;5d976b9e
XZDIO:     410 reads, 24380968 total bytes in 0.421861 secs
```

the relevant line is:

```
D: skip       100644  1 (   0,   0)  1002 /etc/td-agent-bit/td-agent-bit.conf;5d976b9e
```
